File: Codewars/javascript/5kyu_Josephus Permutation.js
function josephus(items,k){

    let myIndex = -1;
    my_arr = [];

    while (items.length > 0) {
        myIndex = (myIndex + k) % items.length;
        my_arr.push(items.splice(items.indexOf(items[myIndex]), 1));
        console.log(my_arr);
        myIndex -= 1;
    }

    return my_arr
}

console.log(josephus([1,2,3,4,5,6,7],3));

File: Codewars/Python/5kyu/5kyu_Josephus Permutation.py
def josephus(items, k):
    num = -1
    my_list = []

    while len(items) > 0:
        num = (num + k) % len(items)
        my_list.append(items[num])
        items.pop(num)
        num -= 1
    return my_list
